static/js/util/util_test.js

```diff
@@ -485,6 +485,10 @@ describe('utility functions', () => {
   describe('formatPrice', () => {
     it('format price', () => {
       assert.equal(formatPrice(20), "$20");
+      assert.equal(formatPrice(20.005), "$20");
+      assert.equal(formatPrice(20.6059), "$20.6");
```

Comment:
In USD prices, we don't have prices that only go to one decimal place. It's either none, or two decimal places (hence "cents" for one-hundredth).